### PR TITLE
982 renaming issue

### LINKED ISSUE #982: [identityManagement] `linkIdentity` method name is misleading
Proposal: replace `linkIdentity` with `requestIdentityLink`.

### Context

As a dApp developer, I would like to have clearer method names.

The practical purpose of `linkIdentity` is to generate a challenge code. An identity is not linked for real until `verifyIdentity` succeeds.

### PR DESCRIPTION
**Pull request by BillyWooo**
rename method/function name

old | new
-- | --
link_identity | create_identity
unlink_identity | remove_identity
verify_identity | verify_identity

### CODE REVIEW

File: pallets/identity-management-mock/src/lib.rs [hunk 1]
```diff
@@ -285,15 +285,15 @@ pub mod pallet {
 
 		/// Link an identity
```
Comment (Kailai-Wang) [inline pallets/identity-management-mock/src/lib.rs]: Can you adjust the comment too? :) (and other places)

File: tee-worker/ts-tests/identity.test.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ describeLitentry('Test Identity', (context) => {
     })
 
     step('link twitter identity', async function () {
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/identity.test.ts]: And the names here :)

**Review by Kailai-Wang — APPROVED** [review]
Looks good, thanks. A few more places to go :)

File: pallets/identity-management-mock/src/lib.rs (hunk 1 above)
Comment (BillyWooo) [reply pallets/identity-management-mock/src/lib.rs]: 😢

File: tee-worker/ts-tests/identity.test.ts (hunk 2 above)
Comment (BillyWooo) [reply tee-worker/ts-tests/identity.test.ts]: 😢

**Comment by jingleizhang** [comment]
LGTM

**Pull request merged**